Added second last and third last period screen

Rits1272 opened the pull request:
Fixes #94

Rits1272 committed:
Added second last and third last period screen

Rits1272 commented:
Please review @sarthology

sahilit commented on src/screens/ProfileScreen/LanguageScreen.js:
```diff
@@ -60,29 +59,15 @@ export default class LanguageScreen extends Component {
-          >
-            please
-          </Text>
-          <Text style={{ fontSize: 25, fontFamily: 'PT-Sans', marginTop: 8, alignSelf: 'center' }}>
```
Solve the bug by fixing the font-family, not by removing it.

sahilit commented on src/screens/ProfileScreen/LanguageScreen.js:
```diff
@@ -60,29 +59,15 @@ export default class LanguageScreen extends Component {
           data={wheelPickerData}
           onItemSelected={this.onItemSelected}
         />
-        <Image source={please_select_your_language} style={styles.cycleText} />
```
Solve the bug by fixing the bug as described in the issue not by removing it.

sahilit commented on src/screens/ProfileScreen/LastPeriodScreen.js:
```diff
@@ -123,65 +135,4 @@ const styles = StyleSheet.create({
   },
 });
 
-const LastPeriodScreen = () => {
```
You need to delete the `LastPeriodScreen` upper part not the lower

sahilit reviewed (commented):
remove the bug fix code. I am assigning it to others

sahilit commented:
This code has removed essential code changes. so it will not be merged.

sahilit commented:
You should read the issue before working on it.
You need to solve the issue not just by deleting the code.
It is not a good way of solving bugs for gaining points.
You should also inform before taking the issue.

Rits1272 commented:
I did not see the issues opened until I finally made the pull request. When I pulled the code, these addressed issues were also popping in my screen, so before working on the assigned issue I first had to solve these neccessary bugs so that finally I can test my own code. Its not that I am just doing it for gaining points @sahilit
